src/main/proto/messages.proto

```diff
@@ -30,7 +30,7 @@ message SamplesSupportingData {
 
 message SparseHistogramEntry {
     double bucket = 1;
-    int32 count = 2;
+    uint64 count = 2;
```

Comment:
Confirmed that [the Protobuf docs say that int32 and uint64 are both varint-encoded](https://developers.google.com/protocol-buffers/docs/encoding#structure), so, yes, wire-compatible.

...unless, perhaps, the sender sends a value not expressible as an int32, in which case who knows what happens on the decoder side. It probably overflows, but I could see it just being undefined behavior.

Reply:
I was somewhat equally puzzled at how that was achieved, however, I believe we are safe because these values should currently never be negative and we're going larger.